modules/security-understanding-container-validation.adoc

```diff
@@ -0,0 +1,157 @@
+
+[NOTE]
+====
+The image manifest is validated only when the manifest digest is known, which is true in the following cases:
+
+. The image is pulled by specifying the SHA256 image digest, for example `quay.io/openshift-release-dev/ocp-release@sha256:ea7ac3ad42169b39fce07e5e53403a028644810bee9a212e7456074894df40f3`.
+
+. The image signature is also being validated. See “Image authenticity validation” for more information.
+====
+
+[id="security-understanding-image-authenticity_{context}"]
+== Image authenticity validation
+
+In addition to validating the integrity of images, {product-title} clusters also validate their authenticity by certifying that the images were approved by a trusted source, in this case Red{nbsp}Hat.
+This validation process includes checking the digital signature of the images.
+
+Cluster administrators can use `ClusterImagePolicy` resources to declare image policy for the cluster.
+
+:FeatureName: Configuring cluster image policies
+include::snippets/technology-preview.adoc[]
+
+.Example `ClusterImagePolicy` resource 1
+[source,yaml]
+----
+kind: ClusterImagePolicy
+metadata:
+  name: mypolicy-0
+spec:
+  scopes:
+  - test0.com
+  policy:
+    rootOfTrust:
+      policyType: FulcioCAWithRekor
+      fulcioCAWithRekor:
+        fulcioCAData: dGVzdC1jYS1kYXRhLWRhdGE=
+        fulcioSubject: <example_input_here> <1>
+        rekorKeyData: dGVzdC1yZWtvci1rZXktZGF0YQ==
+    fulcioSubject:
+      oidcIssuer: https://OIDC.example.com
+      signedEmail: test-user@example.com
+    signedIdentity:
+      matchPolicy: RemapIdentity
+      remapIdentity:
+        prefix: test-remap-prefix
+        signedPrefix: test-remap-signed-prefix
+
+----
+<1> Specifies OIDC issuer and the email of the Fulcio authentication configuration.
+
+.Example `ClusterImagePolicy` resource 2
+[source,yaml]
+----
+kind: ClusterImagePolicy
+metadata:
+  name: mypolicy-1
+spec:
+  scopes:
+  - test0.com <1>
+  - test1.com
+  policy:
+    rootOfTrust:
+      policyType: PublicKey
+      publicKey:
+        keyData: dGVzdC1rZXktZGF0YQ==
+----
+<1> This policy for `test0.com` and the policy from `mypolicy-0` will be appended together.
+
+The previous example resources define two policies, both of which apply to `test0.com` pullspecs, and only one of which applies to `test1.com` pullspecs.
+When image integrity is validated at pull time for images whose pullspecs have configured policies with matching `scopes` values, CRI-O will retrieve any Sigstore signature images associated with the image and check them against the configured policies.
+If signature verification fails for any configured policy, the associated pod will be rejected with <missing_info> conditions in Pod status and events in the associated namespace.
+////
+FIXME: Should "associated pod" be "associated container"?
+FIXME: specific conditions needed for Pod status
+////
+
+[id="security-understanding-trusting-install_{context}"]
+== Trusting an {product-title} installation
```

Comment:
Do you also want to have a section about trusting updates, describing the Cincinnati stream?

It might not be relevant for this article, I don’t know.

Reply:
I can add that section from [this blog](https://www.redhat.com/en/blog/certifiable-software-distribution-process-for-openshift), which I'm using as a reference for this overall PR, if you think it would be valuable to add. Or if it's something that can wait until a v2 of this doc, then I'm okay with that, too.

Reply:
I think we can exclude Cincinnati/OCP updates to ship a v1 of this content, and then we can think through if/how to add something in that space in follow-up work.